lib/index.ts

```diff
@@ -120,7 +124,12 @@ function buildLang (sourceFileName, data) {
         { stdio: [ 'pipe', 1, 2 ] }
     );
 
-    let buffer = subprocess.stdin;
+    if (subprocess.stdin === null) {
+        die('Unable to pipe to elm-format!'); 
+        return false;
+    }
+    
+    let buffer : Writable = subprocess.stdin;
```

Comment:
I think you don't need a space between the name and the colon, perhaps we should format the file with something like prettier

Reply:
Sure, but better to do in another PR.